prac_05/hex_colours.py

```diff
@@ -0,0 +1,17 @@
+"""
+CP1404 - Prac 5
+Program: Hex Colours
+"""
+
+COLOUR_TO_CODE = {"Amethyst": "#9966cc", "Aqua": "#00ffff", "Beige": "#f5f5dc", "Black": "#000000",
+                  "Blue1": "#0000ff", "Brown": "#a52a2a", "Canary": "#ffff99", "Celeste": "#b2ffff",
+                  "Citron": "#9fa91f", "Coral": "#ff7f50"}
+
+colour_name = input("Colour: ")
```

Comment:
You might want to add .title() to this input to handle case-insensitive color names. Users may enter lowercase names, and converting input to title case would ensure it matches the dictionary keys.